src/providers/provider_dota.ts

```diff
@@ -27,8 +26,10 @@ export default class DotaProvider extends Provider {
       const newPatches = [];
 
       // Discard the old patches
+      // eslint-disable-next-line no-plusplus
```

Comment:
Maybe we can actually use another type of loop here...

Reply:
I didn't want to change the lastPatch check so I didn't touch it. We can create an issue and revisit later

Reply:
I moved this to issue #195.